MSSQL: Edit default 'encrypt' setting from false to true (#36634)
grafana/grafana: docs/sources/datasources/mssql.md

| `Authentication` | Authentication mode. Either using SQL Server Authentication or Windows Authentication (single sign on for Windows users).             |
| `User`           | Database user's login/username                                                                                                        |
| `Password`       | Database user's password                                                                                                              |

> | `Encrypt`        | This option determines whether or to which extent a secure SSL TCP/IP connection will be negotiated with the server, default `true`. |

Reviewer: ```suggestion
| `Encrypt`        | This option determines if and to what extent a secure SSL TCP/IP connection will get negotiated with the server. The default value is`true`.  |
```

Reviewer 2: I've added that suggestion @achatterjee-grafana, thanks.
